docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,321 @@ redirect_from:
+
+This page contains an overview of your app in addition to important credentials you'll need later, like the `Signing Secret` under the **App Credentials** header. 
+
+![Basic Information page](../assets/basic-information-page.png "Basic Information page")
+
+Look around, add an app icon and description, and then let's start configuring your app 🔩
+
+---
+
+### Tokens and installing apps
+Slack apps use [OAuth to manage access to Slack's APIs](https://api.slack.com/docs/oauth). When an app is installed, you'll receive a token that the app can use to call API methods. 
+
+There are two token types available to a Slack app: user (`xoxp`) and bot (`xoxb`) tokens. User tokens allow you to call API methods on behalf of users after they install or authenticate the app. There may be several user tokens for a single workspace. Bot tokens are associated with bot users, and are only granted once in a workspace where someone installs the app. The bot token your app uses will be the same no matter which user performed the installation. Bot tokens are the token type that _most_ apps use.
+
+For brevity, we're going to use bot tokens for this guide.
+
+Navigate to the **OAuth & Permissions** on the left sidebar and scroll down to the **Bot Token Scopes** section. Click **Add an OAuth Scope**.
+
+For now, we'll just add one scope: [`chat:write`](https://api.slack.com/scopes/chat:write). This grants your app the permission to post messages in channels it's a member of.
+
+Scroll up to the top of the OAuth & Permissions page and click **Install App to Workspace**. You'll be led through Slack's OAuth UI, where you should allow your app to be installed to your development workspace.
+
+Once you authorize the installation, you'll land on the **OAuth & Permissions** page and see a **Bot User OAuth Access Token**.
+
+![OAuth Tokens](../assets/bot-token.png "Bot OAuth Token")
+
+> 💡 Treat your token like a password and [keep it safe](https://api.slack.com/docs/oauth-safety). Your app uses it to post and retrieve information from Slack workspaces.
+
+---
+
+### Setting up your local project
+With the initial configuration handled, it's time to set up a new Bolt project. This is where you'll write the code that handles the logic for your app.
+
+If you don’t already have a project, let’s create a new one. Create an empty directory:
+
+```shell
+mkdir first-bolt-app
+cd first-bolt-app
+```
+
+<!-- TODO - Do we want to explain how to install the `pyenv` runtime management in a <details> section? -->
+Next, we recommend using a [Python virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) to manage your project's dependencies. This is a great way to prevent conflicts with your system's Python packages. Let's create and activate a new virtual environment:
+
+```shell
+python3 -m venv .venv
```

Comment:
Interesting -- I've never seen an example prepend the `venv` with a `.`. Is there a reason for this?

Reply:
This is a good question and I'm open to the best practice. Since I'm new to Python, I spent some time researching the best practice for virtual environment naming conventions.

In the [official Python Virtual Environment](https://docs.python.org/3/tutorial/venv.html) tutorial they using `.venv` for two reasons:
1. The dot file name keeps the file hidden by most systems and editors
2. It avoids naming collisions with `env` that's a common system-level file/command

> A common directory location for a virtual environment is `.venv`. This name keeps the directory typically hidden in your shell and thus out of the way while giving it a name that explains why the directory exists. It also prevents clashing with .env environment variable definition files that some tooling supports.

I documented this reasoning in the commit message details as well 184c80f912c531995059368648401ef4fda6f8c4

Totally open to the best practice, so I'm 100% open to changing it.

Reply:
I personally don’t use the name .venv (I tend to env or env_*) but it’s a good idea to follow the official document.

Reply:
Interesting. Thanks for taking the time to explain/share this. Learned something new. 🙂

Reply:
Cool, I'll keep this as `.venv` as well. I learned something new myself!

A side note - I'll add `env_*` to the sample app's `.gitignore`.

Reply:
@mwbrooks Thanks! Let's have `env*` in `.gitignore`. Having underscores is just my style.